ansible/roles/oso_zagg_deploy/tasks/main.yml

```diff
@@ -0,0 +1,54 @@
+---
+- block:
+  - name: importing the zagg web image
+    oc_image:
+      registry_url: registry.ops.openshift.com/ops
+      image_name: oso-rhel7-zagg-web
+      image_tag: "{{ oo_environment }}"
+
+  - name: create secrets
+    oc_secret:
+      namespace: default
+      name: monitoring-secrets
+      delete_after: true
+      contents:
+        - path: zagg-server-vars.yml
+          data: "{{ lookup('file', './private_vars/global/zagg-server-vars.yml') }}"
+
+  - name: create the zagg service
+    oc_service:
+      namespace: default
+      name: zagg-service
+      ports:
+      - name: 80-tcp
+        port: 80
+        protocol: TCP
+        targetPort: 8000
+      - name: 443-tcp
+        port: 443
+        protocal: TCP
+        targetPort: 8000
+      selector:
+        oso-rhel7-zagg-web: default
+      service_type: ClusterIP
+
+  - name: checking for existing pods
+    oc_obj:
+      state: list
+      namespace: default
+      kind: pods
+    register: podlist
+
+  - set_fact:
+      zagg_pods_running: "{{ podlist.results | get_running_pods(['oso-rhel7-zagg-web']) }}"
+
+  - name: create app from template
+    oc_process:
+      namespace: default
+      template_name: oso-zagg
+      content: "{{ lookup('file', 'files/zagg.yml') }}"
```

Comment:
not in love with lookup plugin as it runs locally to the host where this is running.  Will not block on this.